instance/static/html/instance/index.html

```diff
@@ -70,16 +66,27 @@
                   | <a href="{{ selected.instance.github_pr_url }}" target="_new">PR#{{ selected.instance.github_pr_number }}</a>
                 </span>
               </li>
+              <li class="instance-server-list">
+                Servers:
+                <span ng-repeat="server in selected.instance.active_server_set">
+                  VM {{server.id}} ({{server.status_name}})
+                </span>
+              </li>
             </ul>
           </div>
           <div class="large-2 columns">
-            <div class="status-icon" ng-if="selected.instance.progress != 'running'">
-              <button class="button reprovision" ng-click="provision(selected.instance)">
+            <div ng-if="selected.instance.is_steady"
+                 ng-attr-class="status-icon status-{{ selected.instance.status }}">
+              <button>
```

Comment:
This is supposed to be `<button class="button reprovision" ng-click="provision(selected.instance)">`

Currently, the Provision button is broken (nothing happens when I click on it).